src/lib/utils/project/GitProjectService.ts

```diff
@@ -462,7 +462,7 @@ export default class GitProjectService {
 
         dependenciesSplitMetadata.push(receiver);
         receivers.push(receiver);
-      } else {
+      } else if (isValidGitUrl(urlOrAddress)) {
```

Comment:
Maybe we should rename `urlOrAddress` to something else now we support Drip Lists?
Something more generic like `dependencyIdentifier` or similar?

Reply:
yeah